proposed/extended-coding-style-guide.md

```diff
@@ -1044,22 +1067,14 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
-## 9. Type Casting
```

Comment:
There was a reference to it in meta-document. Should be checked.

Reply:
Good catch! I updated the meta-doc to contain the new structure. Thanks!